outbound/port/outboud-filterer.ts

```diff
@@ -0,0 +1,6 @@
+import * as O from "fp-ts/Option";
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type OutboundFilterer<T> = {
+  readonly filterArray: (documents: ReadonlyArray<T>) => ReadonlyArray<T>;
+  readonly filter: (document: T) => O.Option<T>;
```

Comment:
Since it's never used right now, do we need it?

Reply:
Removed in [2398f13](https://github.com/pagopa/io-functions-elt/pull/59/commits/2398f13893a44c5a3b27d8855f6851de11d39d16)